src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1664,7 +1664,15 @@ public String getContig() {
 
     /**
      * @return 1-based inclusive start position of the Variant
```

Comment:
Since you are here, I would reformat the javadoc like this:

```
/**
 * Summary sentence.
 *<p>
*    explanation of what it does without loosing time in details of particular edge cases.
 *</p>
* <p>
*   edge case-1
*</p>
* <p>
*   edge case-2
* </p>
*
* @return the text above should had made clear wha is returned... here you report on the possible range of values very briefly.
*/
```

Reply:
```
For example in this case something like this:

/**
  * Returns the (start) position of the variant.
  * <p>
  *   Main blah blah ... 1-based  ... blah blah.
  *</p>
  *<p>
  * For telomeres  blah blah can be 0  or N+1 blah blah
  *</p>
  * @return 0 or greater, never a negative number.
 */
```

Reply:
Here I'm just putting emphasis in the summary sentence (first one finished with .) and the statement in the [at]return tag, the details in the section in between is up to you (that is way I add those "blah" "blah").

That said I think you should be more concise... The programmer don't want to expend to much time to understand what may happen in 0.1% percent of cases.   For example I would not include details on what is said or depicted in the spec, just simply refer to it for further reading/details.